katas/content/marking_oracles/periodic/Solution.qs

```diff
@@ -0,0 +1,29 @@
+namespace Kata {
+    operation Oracle_PeriodicGivenPeriod (x : Qubit[], y : Qubit, p : Int) : Unit is Adj + Ctl {
```

Comment:
Let's swap these two, so that the operation for which the learner had to write code in this task goes first, and the helper copy-pasted from the previous operation goes second (and let's remove comments from both)